scripts/build/TestPlatform.Settings.targets

```diff
@@ -52,6 +52,11 @@
         <GenerateDocumentationFile>false</GenerateDocumentationFile>
       </PropertyGroup>
 
+      <!-- Test projects are not discovered in test window without test container capability -->
+      <ItemGroup>
```

Comment:
This file will get included in product code as well. Shouldn't we add this only for test project?